contrib/inventory/ec2.py

```diff
@@ -435,15 +435,15 @@ def read_settings(self):
 
             aws_access_key_id = config.get('credentials', 'aws_access_key_id')
             aws_secret_access_key = config.get('credentials', 'aws_secret_access_key')
-            aws_security_token = config.get('credentials', 'aws_security_token')
+            aws_session_token = config.get('credentials', 'aws_session_token')
```

Comment:
Simplest backward compatible fix is (untested):

```
aws_session_token = config.get('credentials', 'aws_session_token', config.get('credentials', 'aws_security_token'))
```
I can't find any evidence that boto (as opposed to boto3) supports aws_session_token, so best to support both for people using the session token with boto.

Reply:
Go for it! Problem still persits!